tests/mypy/modules/plugin_fail.py

```diff
@@ -226,3 +226,22 @@ class Config:
 
 inheriting2 = InheritingModel2(x=1, y='c')
 inheriting2.y = 'd'
+
+
+class FieldDefaultTestingModel(BaseModel):
```

Comment:
All the valid cases should go in `success.py`, only the errors should be in this file.

Reply:
Fixed, moving the valid cases to `success.py`, thanks!